hledger-lib/Hledger/Utils.hs

```diff
@@ -186,12 +184,7 @@ readFileOrStdinPortably f = openFileOrStdin f ReadMode >>= readHandlePortably
     openFileOrStdin f m   = openFile f m
 
 readHandlePortably :: Handle -> IO Text
-readHandlePortably h = do
-  hSetNewlineMode h universalNewlineMode
-  menc <- hGetEncoding h
-  when (fmap show menc == Just "UTF-8") $  -- XXX no Eq instance, rely on Show
-    hSetEncoding h utf8_bom
-  T.hGetContents h
+readHandlePortably h = hSetNewlineMode h universalNewlineMode *> TIO.hGetContents h
```

Comment:
I think >> might be a little more idiomatic and obvious in this codebase

Reply:
Sure thing.